src/build/arg/mod.rs

```diff
@@ -159,14 +159,15 @@ impl<'help> Arg<'help> {
     ///
     /// # Examples
     ///
-    /// ```ignore
-    /// # #[macro_use]
-    /// # extern crate clap;
-    /// # use clap::Arg;
+    /// ```no_run
+    /// #[cfg(feature = "yaml")]
     /// # fn main() {
+    /// # use clap::{Arg, load_yaml};
     /// let yml = load_yaml!("arg.yml");
     /// let arg = Arg::from_yaml(yml);
```

Comment:
This line now fails because `load_yaml!` seems to return a different type:

```

error[E0308]: mismatched types
 --> src/build/arg/mod.rs:167:26
  |
7 | let arg = Arg::from_yaml(yml);
  |                          ^^^ expected struct `linked_hash_map::LinkedHashMap`, found enum `yaml_rust::yaml::Yaml`
  |
  = note: expected type `&linked_hash_map::LinkedHashMap<yaml_rust::yaml::Yaml, yaml_rust::yaml::Yaml>`
             found type `&yaml_rust::yaml::Yaml`

```